### ISSUE TITLE
Look around: info about `unseen` tiles from memory

### ISSUE THREAD

**Issue by Brambor** [CleverRaven/Cataclysm-DDA#87467]
### Is your feature request related to a problem? Please describe.

<img width="641" height="384" alt="Image" src="https://github.com/user-attachments/assets/19d006a6-1ee4-4f7b-8ef0-23de84b99f50" />

My character remembers there is a `field` with a `rose bush`. This is depicted by drawing the texture of a `rose bush`. But this is not reflected in the text info given.

### Solution you would like.

I would like to see this kind of info from memory.
<img width="641" height="388" alt="Image" src="https://github.com/user-attachments/assets/fd5210e4-5cf9-41ff-b9f2-07f92f5815d0" />

Namely:
 - What floor is there?
 - What terrain is there?
 - What furniture is there?

Things to NOT show:
 - lighting
 - items
 - ... ?

I am not asking about memorising more or showing the state of the `rose bush` (whether it was harvested or not). Just an explanation about what that tile is.

I would especially use this when running from danger in a forest.

**Comment by dobbry-vechur** [comment]
I feel like implementing this properly would require some sort of short-term memory abstraction, fading over time.

**Comment by Brambor** [comment]
I don't want to remember anything new; the character already remembers it. This is not a discussion about what the character remembers, but how it is presented to the player.

It is more like a wiki or atlas. It is already implemented that the character remembers there is a `rose bush`. I just want to present the memory not only visually (picture of the bush), but also textually (textual explanation of the bush).

I expect that the implementation of memory stores `id` of the `field` and `id` of the `rose bush`. So here I expect to print info about these `id`s that you already get when you look at them.

**Comment by dobbry-vechur** [comment]
Yeah, but I mean, that would not make much sense to remember some particular rose bush that you last saw a year ago.

**Comment by Brambor** [comment]
Might be true. But this issue is not about this. This was already heavily discussed.

I cannot recognise from the picture that it is a rose bush. I want the game to explicitly write that.

If I edited the textures and wrote "rose bush" into the sprite of it, I would recognise it.

I guess this is an accessibility issue.

**Comment by PatrikLundell** [comment]
I agree with @Brambor, especially since tiles are reused, so you can't actually tell which of the several things the graphic can represent that's actually being represented. Several types of fruit bearing plants (bushes and trees) use the same graphics in Ultica, but I may actually only want some of the possible versions, which is annoying when foraging and the next plant you may want to go to is obscured so you can't check its type.

It can be noted that the memory can show things that aren't actually there anymore, because things have changed since you last visited the place (such as temporary traders). That's perfectly fine.

**Comment by Brambor** [comment]
I am implementing this currently. Hopefully, I will have a proof of concept today.
